src/adapters/bidderFactory.js

```diff
@@ -224,6 +225,12 @@ export function newBidder(spec) {
       requests.forEach(processRequest);
 
       function processRequest(request) {
+        // check for undefined or null request.options
+        for (var member in request.options) {
+          if (!request.options[member]) {
```

Comment:
What if someone wanted to override `withCredentials: false`?

Is there any particular reason you're trying to delete null & undefined here?

Reply:
It is possible to overwrite the `withCredentials` by setting it in the options object since I have used the change that you suggested. I can add that to the tests if necessary. 

I added the check for null for the sake of completeness. If someone is using logic to set this value and somehow passes a null or undefined then that member variable gets removed from the replacement array.

Reply:
Ah yeah, that was dumb my bad.

Reply:
I don't think so... I just ran it myself:

```
    it('should allow falsey options in the overrides', () => {
      const bidder = newBidder(spec);
      const url = 'test.url.com';
      const data = { arg: 2 };
      const options = { withCredentials: false }
      spec.isBidRequestValid.returns(true);
      spec.buildRequests.returns({
        method: 'POST',
        url: url,
        data: data,
        options: options
      });

      bidder.callBids(MOCK_BIDS_REQUEST);

      expect(ajaxStub.calledOnce).to.equal(true);
      expect(ajaxStub.firstCall.args[0]).to.equal(url);
      expect(ajaxStub.firstCall.args[2]).to.equal(JSON.stringify(data));
      expect(ajaxStub.firstCall.args[3]).to.deep.equal({
        method: 'POST',
        contentType: 'text/plain',
        withCredentials: false        // The test succeeds if this is true
      });
    });
```

Pretty sure you're deleting all false-y properties from the options object, so they don't override properly.